fv3core/stencils/remap_profile.py

```diff
@@ -559,39 +538,25 @@ def __init__(
         domain: Tuple[int, int, int] = (i_extent, j_extent, km)
         domain_extend: Tuple[int, int, int] = (i_extent, j_extent, km + 1)
 
-        self._set_values_stencil = FrozenStencil(
-            func=set_vals,
+        self._set_initial_values_stencil = FrozenStencil(
```

Comment:
Should this one be named `_set_intial_values` to be consistent with the other two, `_apply_constraints` and `_set_interpolation_coefficients` not having `stencil` in the name?